modules/social.py

```diff
@@ -133,30 +133,30 @@ def __init__(self, bot):
             'Most likely',
         ]
 
-        self.triviaTrophyIndex = {
-            0: 'no-active-trophies',
-            1: 'trivia-bronze-1',
-            2: 'trivia-bronze-2',
-            3: 'trivia-bronze-3',
-            4: 'trivia-silver-1',
-            5: 'trivia-silver-2',
-            6: 'trivia-silver-3',
-            7: 'trivia-gold-1',
-            8: 'trivia-gold-2',
-            9: 'trivia-gold-3',
-        }
-        self.triviaTrophyEmotes = {
-            0: '',
-            1: '<:triviabronze1:1194031669498351656>',
-            2: '<:triviabronze2:1194031670421110945>',
-            3: '<:triviabronze3:1194031672690229338>',
-            4: '<:triviasilver1:1194031683251482696>',
-            5: '<:triviasilver2:1194031687810699366>',
-            6: '<:triviasilver3:1194031688792154234>',
-            7: '<:triviagold1:1194031674053382216>',
-            8: '<:triviagold2:1194031676649652305>',
-            9: '<:triviagold3:1194031677715005490>',
-        }
+        self.triviaTrophyIndex = [
```

Comment:
Looks good! Only other suggestion I had was still combining these is that there is the option to combine these two arrays into a single one, but it is not a priority 
```
self.trophyData = [
    ('no-active-trophies', ''),
    ('trivia-bronze-1', '<:triviabronze1:1194031669498351656>'),
    ...
]

Reply:
If you do this, just make sure to replace `self.triviaTrophyIndex.index(t)` with `[n for (n, _) in self.trophyData].index(t)`